src/masternodeman.cpp

```diff
@@ -973,7 +973,7 @@ void CMasternodeMan::ProcessMessage(CNode* pfrom, std::string& strCommand, CData
         // make sure the vout that was signed is related to the transaction that spawned the Masternode
         //  - this is expensive, so it's only done once per Masternode
         if (!obfuScationSigner.IsVinAssociatedWithPubkey(vin, pubkey)) {
-            LogPrintf("dsee - Got mismatched pubkey and vin\n");
+            LogPrint("masternode","dsee - Got mismatched pubkey and vin\n");
```

Comment:
post-merge: This should have been left as `LogPrintf()` as it results in a `Misbehaving` message.